source/style/content.css

```diff
@@ -6,7 +6,9 @@ Global
 
 /* Use system fonts */
 body {
-	font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol' !important;
+	font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
```

Comment:
Seems like prettier decided to wrap some stuff, can you undo this? :) I'll comment just here but there are several places that prettier has acted on.